pymeasure/instruments/keithley/keithley2000.py

```diff
@@ -58,10 +58,10 @@ class Keithley2000(Instrument, KeithleyBuffer):
     mode = Instrument.control(
         ":CONF?", ":CONF:%s",
         """ A string property that controls the configuration mode for measurements,
-        which can take the values: :code:'current' (DC), :code:'current ac',
-        :code:'voltage' (DC),  :code:'voltage ac', :code:'resistance' (2-wire),
-        :code:'resistance 4W' (4-wire), :code:'period', :code:'frequency',
-        :code:'temperature', :code:'diode', and :code:'frequency'. """,
+        which can take the values: ``current`` (DC), ``current ac``,
```

Comment:
Why do you use the double backquotes?
For sphinx ":code:`value = 7`" will render the part "value = 7" as a code snippet. Therefore `:code:` might be correct here. Even the Apostrophes are right, as there are strings, but backticks are missing:
```suggestion
        which can take the values: :code:`'current'` (DC), :code:`'current ac'`,
```

Reply:
Double backticks and `:code:` are equivalent as far as I can tell: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

I usually use double backticks, easier to type, and b/c the code lines weren't rendering that is what I used. I'm fine moving it to `:code:`, should that happen?

Reply:
> Double backticks and :code: are equivalent

Ah, now I understand. Thanks for telling, it's good to know.